lib/omnibus/packagers/solaris.rb

```diff
@@ -35,11 +35,10 @@ class Packager::Solaris < Packager::Base
       copy_file("#{project.package_scripts_path}/postinst", '/tmp/pkgmk/postinstall')
       copy_file("#{project.package_scripts_path}/postrm", '/tmp/pkgmk/postremove')
 
-      Dir.chdir(staging_dir) do
-        shellout! "pkgmk -o -r #{install_dirname} -d /tmp/pkgmk -f /tmp/pkgmk/Prototype"
-        shellout! "pkgchk -vd /tmp/pkgmk #{project.package_name}"
-        shellout! "pkgtrans /tmp/pkgmk /var/cache/omnibus/pkg/#{package_name} #{project.package_name}"
-      end
+      shellout! "pkgmk -o -r #{install_dirname} -d /tmp/pkgmk -f /tmp/pkgmk/Prototype"
+      shellout! "pkgchk -vd /tmp/pkgmk #{project.package_name}"
+      abs_package_path = File.absolute_path(package_path) 
+      shellout! "pkgtrans /tmp/pkgmk #{abs_package_path} #{project.package_name}"
```

Comment:
We shouldn't need to make this `File.absolute_path` call. If `package_path` needs to be expanded it should be done in `Omnibus::Packager::Base`: https://github.com/chef/omnibus/blob/c9937e6d3729aa72bf45c0bd77a6ff9d4c2c5d09/lib/omnibus/packagers/base.rb#L159-L168

So line 41 should just look like:

```ruby
shellout! "pkgtrans /tmp/pkgmk #{package_path} #{project.package_name}"
```